src/libtomahawk/accounts/Account.cpp

```diff
@@ -153,7 +153,7 @@ Account::removeFromConfig()
 void
 Account::testConfig()
 {
-    emit configTestResult( ConfigTestResultSuccess );
+    emit configTestResult( ConfigTestResultSuccess, "" );
```

Comment:
Don't ever create QStrings with "", create them with QString(). I'd prefer having this a default parameter though, instead of passing QString() manually several times